src/wtc-utility-helpers.js

```diff
@@ -1,95 +1,71 @@
-var utilities = {};
-
 /**
  * Generate a random float number max and min.
- * 
+ *
  * ```javascript
  * utilities.floatRandomBetween(-10, 20); // 12.513
  * ```
- * 
- * @public
+ *
  * @param {number} min Minimum value.
  * @param {number} max Maximum value.
  * @return {number} Random number.
  */
-utilities.floatRandomBetween = function(min, max) {
+export const floatRandomBetween = (min, max) => {
   return Math.random() * (max - min + 1) + min;
```

Comment:
FWIW this algorithm is wrong. We should remove the `+ 1`

Reply:
Done